presto-main/src/test/java/com/facebook/presto/sql/planner/TestDomainTranslator.java

```diff
@@ -158,6 +158,27 @@ public void testRoundTrip()
     }
 
     @Test
+    public void testInOptimization()
```

Comment:
add a test case that includes multiple ranges with negatable single values, and some with of those ranges not having any negatable single values